ObjCVideoQuickstart/ViewController.m

```diff
@@ -0,0 +1,314 @@
+//
+//  ViewController.m
+//  ObjCVideoQuickstart
+//
+//  Copyright © 2016 Twilio, Inc. All rights reserved.
+//
+
+#import "ViewController.h"
+#import "Utils.h"
+
+#import <TwilioVideo/TwilioVideo.h>
+
+@interface ViewController () <UITextFieldDelegate, TVIParticipantDelegate, TVIRoomDelegate>
+
+// Configure access token manually for testing in `ViewDidLoad`, if desired! Create one manually in the console.
+@property (nonatomic, strong) NSString *accessToken;
+
+#pragma mark Video SDK components
+
+@property (nonatomic, strong) TVIVideoClient *client;
+@property (nonatomic, strong) TVIRoom *room;
+@property (nonatomic, strong) TVILocalMedia *localMedia;
+@property (nonatomic, strong) TVICameraCapturer *camera;
+@property (nonatomic, strong) TVILocalVideoTrack *localVideoTrack;
+@property (nonatomic, strong) TVILocalAudioTrack *localAudioTrack;
+@property (nonatomic, strong) TVIParticipant *participant;
+
+#pragma mark UI Element Outlets and handles
+
+@property (weak, nonatomic) IBOutlet UIView *remoteView;
+@property (weak, nonatomic) IBOutlet UIView *previewView;
+@property (weak, nonatomic) IBOutlet UIView *connectButton;
+@property (weak, nonatomic) IBOutlet UIButton *disconnectButton;
+@property (weak, nonatomic) IBOutlet UILabel *messageLabel;
+@property (weak, nonatomic) IBOutlet UITextField *roomTextField;
+@property (weak, nonatomic) IBOutlet UIButton *micButton;
+@property (weak, nonatomic) IBOutlet UILabel *roomLabel;
+@property (weak, nonatomic) IBOutlet UILabel *roomLine;
+
+@end
+
+@implementation ViewController
```

Comment:
Use pragma marks to separate `UIViewController`, `Public`, and `Private` methods?